src/main/kotlin/com/zepben/evolve/database/sqlite/cim/tables/iec61968/common/TableCurves.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.tables.iec61968.common
+
+import com.zepben.evolve.database.sqlite.cim.tables.iec61970.base.core.TableIdentifiedObjects
+
+abstract class TableCurves : TableIdentifiedObjects() {
+
+    override val name: String = "curve"
```

Comment:
todo: Should not be settings the name on an abstract table. This applies to all your new abstract tables.

Reply:
removed from all